infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationMetadataManager.java

```diff
@@ -596,14 +598,15 @@ public void setReplicationStatusTable(String clusterId, long remainingEntries, S
                 txn.commit();
             }
             
-            log.debug("syncStatusPoller :: Log Entry status set to ONGOING, clusterId: {}, remainingEntries: {}, " +
-                            "snapshotSyncInfo: {}", clusterId, remainingEntries, snapshotStatus);
+            log.debug("syncStatusPoller :: Log Entry status set to {}, clusterId: {}, remainingEntries: {}, " +
+                            "snapshotSyncInfo: {}", currentStatus, clusterId, remainingEntries, snapshotStatus);
         } else if (type == SyncType.SNAPSHOT) {
 
             SnapshotSyncInfo currentSnapshotSyncInfo;
             if (snapshotStatus == null){
                 log.warn("syncStatusPoller [snapshot] :: previous status is not present for cluster: {}", clusterId);
-                currentSnapshotSyncInfo = SnapshotSyncInfo.newBuilder().build();
+                currentSnapshotSyncInfo = SnapshotSyncInfo.newBuilder().setStatus(SyncStatus.UNAVAILABLE).build();
```

Comment:
In this case it should be NOT_STARTED... for log entry it is unavailable only because it is an error state, if it's already in log_entry it should've performed full sync and have this state present. However, in this case you're about to start snapshot there isn't a direct implication that a previous snapshot is present.

Reply:
Yes, that makes more meaning. I have updated the status to NOT_STARTED.